bundles/com.salesforce.bazel.eclipse.core/src/com/salesforce/bazel/eclipse/core/util/trace/TraceTree.java

```diff
@@ -102,7 +105,8 @@ private static SpanNode createNode(Span span, long rootDurationNanos) {
         }
 
         var durationNanos = span.getDuration(TimeUnit.NANOSECONDS);
-        return new SpanNode(span.getName(), durationNanos, percentage(durationNanos, rootDurationNanos), children);
+        var startTimeEpocMilli = span.getStartTimeEpocMilli();
```

Comment:
```suggestion
        long startTimeEpocMilli = span.getStartTimeEpocMilli();
```
nit: this variable could be explicitly set to `long` for readability.

Reply:
same for the `durationNanos` variable above this one

Reply:
@AdwaitChavan I disagree. `var` is perfectly readable. This project actually has code clean-ups on save in Eclipse, which will ensure `var` is used where possible.